ethereum/contracts/bridgehub/bridgehub-interfaces/IBridgehubForProof.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.13;
+
+import "./IBridgehubBase.sol";
+import "../../common/Messaging.sol";
+
+interface IBridgehubForProof is IBridgehubBase {
```

Comment:
It seems like this interface is never used

Reply:
removed